frontend/src/services/syncService.js

```
@@ -0,0 +1,309 @@
+          }));
+        });
+      
+      default:
+        return remoteItem;
+    }
+  }
+
+  async pushLocalChange(tableName, localItem) {
+    try {
+      const itemToSync = { ...localItem };
+      delete itemToSync._pendingSync;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)